api/cas-server-core-api-authentication/src/main/java/org/apereo/cas/authentication/MfaCredential.java

```diff
@@ -0,0 +1,25 @@
+package org.apereo.cas.authentication;
+
+/**
+ * This interface is used by credentials that are created by an Mfa provider.
+ *
+ * @author Travis Schmidt
+ * @since 5.3.4
+ */
+public interface MfaCredential {
+
+    /**
+     * Method returns the provider id of the configured provider that created this credential.
+     *
+     * @return - the provider id
+     */
+    String getProviderId();
```

Comment:
This needs to be renamed to be more explicit because Credential in master presents a property for `source` and it would be confusing to have both source and providerId in the same class. Something slightly more verbose like MultifactorProviderId would do, I think.

Reply:
Sounds good